tests/recipient_validation/test_phone_number.py

```diff
@@ -361,15 +362,16 @@ def test_validates_against_guestlist_of_international_phone_numbers(recipient_nu
         ("447900900123", "07900 900123"),  # UK
         ("20-12-1234-1234", "+20 12 12341234"),  # Egypt
         ("00201212341234", "+20 12 12341234"),  # Egypt
-        ("1664 0000000", "+1 664-000-0000"),  # Montserrat
+        ("16644918766", "+1 664-491-8766"),  # Montserrat
         ("7 499 1231212", "+7 499 123-12-12"),  # Moscow (Russia)
         ("1-202-555-0104", "+1 202-555-0104"),  # Washington DC (USA)
-        ("+23051234567", "+230 5123 4567"),  # Mauritius
+        ("+2304031001", "+230 403 1001"),  # Mauritius
```

Comment:
Fixes numbers that were in fact not valid for the country

Reply:
Would be good to gather all changes where you change a number to be actually a valid number into one commit with commit message like your comment here.